apps/web/src/components/Dashboard/Templates.tsx

```diff
@@ -74,34 +190,145 @@ export function TemplatesContent({
                 className="hover:bg-orange-300/10 dark:hover:bg-orange-300/10 border-b border-white/5"
                 key={template.templateID}
               >
+                <TableCell>
+                  <TooltipProvider>
+                    <Tooltip>
+                      <TooltipTrigger asChild>
+                        <Button
+                          variant="ghost"
+                          size="icon"
+                          onClick={() => {
+                            setCurrentTemplate(template)
```

Comment:
preventing event propagation is probably a good idea inside a dynamic table like this

Reply:
it should be alright, no?

Reply:
i think it's good practice for nested elements even if it's currently alright, might come back to bite you when updating event handlers of parent elements.

Reply:
It isn't in form, so it shouldnt be problem right?

Reply:
you're thinking of `preventDefault()`, the `preventPropagation()` is to avoid triggered other parent `onClick`s

Reply:
have added it, but keep in mind there are no event listeners in parent

Reply:
I am not sure if I am not missing something, but as Mish says, there are no parent elements with a click handler and I wouldn't expect there to be any. Is there a reason why to put it here? Wouldn't you put it everywhere then?

Reply:
In this case all the `<Table...` elements and im not saying there are currently click handlers, i was saying it could avoid unexpected behavior were there to be event handlers added in the future.

Reply:
don't worry about it, the button is removed from there now and the table we'll be replacing anyways with a tan-stack one